src/sentry/static/sentry/app/components/events/groupingInfo/groupingVariant.tsx

```diff
@@ -106,9 +155,9 @@ class GroupVariant extends React.Component<Props, State> {
           <ContributionIcon isContributing={isContributing} />
           {t('By')}{' '}
           {variant.description
-            .split(' ')
+            ?.split(' ')
```

Comment:
Can `variant` be null?

Reply:
There's this one edge case where the server can send `null` description.